bl-kernel/admin/themes/booty/css/bludit.css

```
@@ -773,3 +775,13 @@ select:focus,
 
 
 
+nt;
```

Stylelint (17.6.0)

[error] Unknown word nt (CssSyntaxError)

(CssSyntaxError)